storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/introspection/StorIOColumnMeta.java

```diff
@@ -41,6 +41,16 @@ public boolean isMethod() {
         return element.getKind() == ElementKind.METHOD;
     }
 
+    public String getRealElementName() {
+        if (elementName.startsWith("get") && Character.isUpperCase(elementName.charAt(3))) {
+            return elementName.substring(3).toLowerCase();
```

Comment:
what about upper case symbol in the middle of the world, will we support names in camel case?